make/common/NativeCompilation.gmk

```diff
@@ -447,7 +447,7 @@ define SetupCompileNativeFileBody
     $$($1_OBJ): $$($1_OBJ_DEPS) | $$($$($1_BASE)_BUILD_INFO)
 	$$(call LogInfo, Compiling $$($1_FILENAME) (for $$($$($1_BASE)_BASENAME)))
 	$$(call MakeDir, $$(@D))
-        ifneq ($(TOOLCHAIN_TYPE), microsoft)
+        ifeq ($(findstring $(TOOLCHAIN_TYPE), microsoft clcl), )
```

Comment:
I know `findstring` is used frequently for this construct in the makefile, but I prefer `filter` in cases like this as it only matches complete words.